### ISSUE TITLE
Issue with reset tasks

### ISSUE THREAD

**Issue by Mrpuppyz** [meeb/tubesync#300]
I found an issue when trying to use the reset task button at the bottom of the task page and also using the command line to reset task both fail. I opened two terminals and ran the follow logs in one terminal. Then I opened another terminal and ran the reset tasks command and watched as it reset a a few channels. While it was still resetting the tasks in the other terminal it began to index the channels that have already been reset. Eventually a channel that had been reset started to write the changes from the indexing at the same time as a reset task command was trying to write changes causing the reset tasks terminal to fail to reach database. I was using the ordinary configuration with the built-in MySQL light file. I believe the easiest way to fix this problem would be to not let other channes start to index until all of the sources have been reset.

**Comment by meeb** [comment]
Do you mean SQLite? Then yes this could cause a lock issue probably. The "reset tasks" was always a pretty awful hack. I'll check it though, thanks.

**Comment by Mrpuppyz** [comment]
Yes I did mean SQLite. Thanks for all you do with this application. I can see how challenging it can be to try to keep up with YouTube in the ways they change things all the time.

**Comment by meeb** [comment]
This seems resolved for now so I'll close it. Please open a new issue if you continue to experience problems.
